posts/2022-08-04-jakarta-core-profile-22009-beta.adoc

```diff
@@ -0,0 +1,343 @@
+
+MicroProfile OpenAPI 3.1 is a minor release that includes the following updates:
+
+* Some Jakarta Bean Validation annotations will now be read and their restrictions will be added to object schemas.
+* Most annotations now include an `extension` attribute.
+* `@SecurityRequirementsSet` can now be used to document operations where authentication is optional or where more than one authentication mechanism is required.
+
+To use MicroProfile OpenAPI 3.1, add the `mpOpenAPI-3.1` feature to your server.xml:
+[source,xml]
+----
+<featureManager>
+    <feature>mpOpenAPI-3.1</feature>
+</featureManager>
+----
+
+Next, deploy your rest application and then visit the `/openapi` endpoint to view the OpenAPI documentation, or visit `/openapi/ui` for a user interface where you can browse through it. You can enhance the documentation by using annotations to add things like textual descriptions to each method. The annotations are available as a maven dependency:
+
+[source, xml]
+----
+<dependency>
+    <groupId>org.eclipse.microprofile.openapi</groupId>
+    <artifactId>microprofile-openapi-api</artifactId>
+    <version>3.1-RC2</version>
+</dependency>
+----
+
+For more information about MicroProfile OpenAPI, see the following links:
+
+* link:{url-prefix}/guides/microprofile-openapi.html[Open Liberty MicroProfile OpenAPI guide]
+* link:{url-prefix}/docs/latest/documentation-openapi.html[Open Liberty OpenAPI documentation]
+* link:https://download.eclipse.org/microprofile/microprofile-open-api-3.1-RC2/apidocs/[MicroProfile OpenAPI 3.1 Javadoc]
+
+[#password]
+== Password Utilities 1.1 
+
+The new `passwordUtilities-1.1` feature no longer starts the `federatedRegistry-1.0` feature or the Jakarta Connectors feature. 
+Stand-alone user registries are no longer forcefully federated, which sometimes results in slightly different behavior. The password utilities APIs can now be used in core edition.
+
+To enable Password Utilities 1.1, add the `passwordUtilities-1.1` feature to the list of features in your server.xml file:
+
+[source, xml]
+----
+<featureManager>
+    <feature>passwordUtilities-1.1</feature>
+</featureManager>
+----
+
+For more information about the Password Utilities, check out the link:{url-prefix}/docs/latest/reference/feature/passwordUtilities.html[feature documentation].
+
+[#run]
+=== Try it now 
+
+To try out these features, just update your build tools to pull the Open Liberty All Beta Features package instead of the main release. The beta works with Java SE 18, Java SE 17, Java SE 11, and Java SE 8, however, the Jakarta EE 10 features are only compatible with Java SE 11 and above.
```

Comment:
```suggestion
To try out these features, just update your build tools to pull the Open Liberty All Beta Features package instead of the main release. The beta works with Java SE 18, Java SE 17, Java SE 11, and Java SE 8, however, the Jakarta EE 10 features are only compatible with Java SE 11 and later.
```

That seems to be the general edit that the doc people suggest.

Reply:
updated